homeassistant/components/konnected.py

```diff
@@ -119,6 +128,35 @@ def device_discovered(service, info):
                             " but not specified in configuration.yaml",
                             discovered.device_id)
 
+    def manual_discovery(event):
+        """Init devices on the network with manually assigned addresses."""
+        specified = list(filter(  # list of devices where host and port is set
```

Comment:
We prefer a list comprehension.

Reply:
Good catch. You're right, this is much better. Sorry, I'm still relatively new to Python.